axelor-production/src/main/resources/domains/AppProduction.xml

```diff
@@ -34,6 +34,8 @@
 		<boolean name="manageOutsourcing" title="Manage outsourcing"/>
 		<boolean name="manageMpsCharge" title="Manage Master Production Scheduling Charge"/>
 		<boolean name="enableToolManagement" title="Enable Tool Management"/>
+		
+		<boolean name="isAutomaticallyPlanified" title="Automatically planified after fusion"></boolean>
```

Comment:
I know it's a detail, but proper practice is to end the tags with /> instead of ></tag> when there is no content.